core/src/components/alert/alert.scss

```diff
@@ -150,6 +150,16 @@
   height: 100%;
 }
 
+
+// Alert Button: Disabled
+// --------------------------------------------------
+
+[disabled] .alert-button-inner {
+  cursor: default;
+  opacity: .3;
```

Comment:
Please use `opacity: .5;` as it is the style commonly used for other disabled buttons.

Reply:
Done 🙂